react-components/src/actions/user/__test__/actions.test.js

```diff
@@ -17,6 +17,9 @@ const setCartCookie = jest.fn();
 const setUserCookie = jest.fn();
 const dispatch = jest.fn();
 
+// avoid console errors logged during testing
+console.error = jest.fn();
```

Comment:
I would rather add the `--silent` parameter to the `jest` call than mocking the `console.error` calls here. This has the advantage that you can easily decide whether or not you want to print the console statements. Sometimes they are useful.

Reply:
The problem with `--silent` is that it supresses all console logs including the warnings that may indicate an issue (such the ones we fixed here).